wasi/wasi.go

```diff
@@ -82,9 +84,60 @@ type API interface {
 	// See https://en.wikipedia.org/wiki/Null-terminated_string
 	ArgsSizesGet(ctx wasm.HostFunctionCallContext, resultArgc, resultArgvBufSize uint32) Errno
 
-	// TODO: EnvironGet(ctx api.hostFunctionCallContext, environ, environBuf uint32) Errno
+	// EnvironGet is the WASI function that reads environment variables.
+	//
+	// There are two parameters. Both are offsets in api.HostFunctionCallContext Memory. If either are invalid due to
```

Comment:
I think this is a drift from @codefromthecrypt as `api` doesn't exist :D

Reply:
indeed